pyproject.toml

```
@@ -34,4 +34,4 @@ test-command = "pytest {project}/test/test_array.py" # same limitation as conda-
 skip = ["*-win32", "pp*"]
 test-command = ""
 before-all = "bash {package}/scripts/build-ocl-windows.sh"
-before-build = "python configure.py --cxxflags=-ID:/a/pyopencl/pyopencl/OpenCL-Headers/install/include --ldflags=\"/LIBPATH:C:/Program Files/OpenCL-ICD-Loader/lib\""
+before-build = "python configure.py --cxxflags=-ID:/a/pyopencl/pyopencl/OpenCL-Headers/install/include --cl-lib-dir=\"C:/Program Files/OpenCL-ICD-Loader/lib\""
```

Use `--cl-inc-dir` too?
```suggestion
before-build = "python configure.py --cl-inc-dir=D:/a/pyopencl/pyopencl/OpenCL-Headers/install/include --cl-lib-dir=\"C:/Program Files/OpenCL-ICD-Loader/lib\""
```

---

Thanks! I thought about it, but, in retrospect, I would have guessed wrong in how to do it. :) Not that it's not obvious in hindsight. I just don't know Windows.